src/components/Tabs/Tabs.module.scss

```diff
@@ -0,0 +1,16 @@
+.tabButton {
+  background: white;
+  border: none;
+  padding: 14px 9px 9px;
+  font-size: 0.8em;
+  border-top-right-radius: 3px;
+  border-top-left-radius: 3px;
+  cursor: pointer;
+}
+
+.active {
+  font-weight: bold;
+  border: 1px solid #d4d4d5;
+  border-bottom: none;
+  background: white;
```

Comment:
I think we can use color variables to maintain the consistency in colors throughout the app